docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4215,6 +4215,10 @@ Leave a swarm
     Content-Length: 0
     Content-Type: text/plain; charset=utf-8
 
+**Query parameters**:
+
+- **force** - Force leave swarm, even if this is the last manager.
```

Comment:
Can you mention this is a Boolean, and that it takes values (0/1, false/true), like we do on the other Boolean query params?